addons/binding/org.openhab.binding.seneye/src/main/java/org/openhab/binding/seneye/service/SeneyeService.java

```diff
@@ -0,0 +1,186 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.openhab.binding.seneye.service;
+
+import java.io.IOException;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.commons.httpclient.DefaultHttpMethodRetryHandler;
+import org.apache.commons.httpclient.HttpClient;
+import org.apache.commons.httpclient.HttpException;
+import org.apache.commons.httpclient.HttpStatus;
+import org.apache.commons.httpclient.methods.GetMethod;
+import org.apache.commons.httpclient.params.HttpMethodParams;
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.Gson;
+
+/**
+ * The {@link SeneyeService} handles the connection to the Seneye API
+ *
+ * @author Niko Tanghe
+ */
+
+public class SeneyeService {
+
+    private static Logger logger = LoggerFactory.getLogger(SeneyeService.class);
+
+    private long refreshInterval;
+    private int retry;
+
+    private String aquariumname;
+    private String userName;
+    private String password;
+    private String seneyeId;
+
+    private boolean isInitialized;
+
+    private final Gson gson;
+
+    public SeneyeService() {
+        this("", "", "", 120000);
```

Comment:
Why are you using a hardcoded refresh interval here?

Reply:
removed this constructor